frontend/packages/core/src/AppLayout/search.tsx

```diff
@@ -153,26 +228,44 @@ const SearchField: React.FC = () => {
     setInputValue("");
   };
 
+  const handleOpen = () => {
+    setOpen(!open);
+  };
+
+  const handleClose = () => {
+    setOpen(false);
+  };
+
   return (
-    <Container>
-      <Autocomplete
-        autoComplete
-        freeSolo
-        selectOnFocus
-        size="small"
-        inputValue={inputValue}
-        renderInput={Input}
-        renderOption={renderResult}
-        onInputChange={onInputChange}
-        open={showOptions}
-        onOpen={onOptionsOpen}
-        onClose={onOptionsClose}
-        options={options}
-        filterOptions={filterResults}
-        getOptionLabel={x => x.label}
-        ListboxProps={{ style: { backgroundColor: fade(theme.palette.secondary.main, 0.75) } }}
-      />
-    </Container>
+    <Grid container alignItems="center">
+      {open ? (
+        <ClickAwayListener onClickAway={handleClose}>
+          <Autocomplete
+            autoComplete
+            selectOnFocus
+            size="small"
+            inputValue={inputValue}
+            renderInput={Input}
+            renderOption={renderResult}
+            onInputChange={onInputChange}
+            open={showOptions}
+            onOpen={onOptionsOpen}
+            onClose={onOptionsClose}
+            options={options}
+            filterOptions={filterResults}
+            getOptionLabel={x => x.label}
+            PopperComponent={renderPopper}
+            popupIcon={<CustomCloseIcon/>}
+            forcePopupIcon={showOptions ? true: false}
+            noOptionsText='No results found'
```

Comment:
noticed a styling bug when there weren't result options, which is due to the paper component having a border.
<img width="591" alt="Screen Shot 2020-12-04 at 11 09 56 AM" src="https://user-images.githubusercontent.com/39421794/101186581-4686eb80-3621-11eb-96e0-379df9e405b3.png">

removing the `freeSolo` prop (which we don't have a use for since user's are bound to provided options) and adding `noOptionsText` is one way to [solve](https://github.com/mui-org/material-ui/issues/18985) it. an alternative is conditionally rendering the paper border. lmk if you have a preference

<img width="591" alt="Screen Shot 2020-12-04 at 11 13 33 AM" src="https://user-images.githubusercontent.com/39421794/101186994-c6ad5100-3621-11eb-9d2d-db93882b6026.png">